cfme/tests/infrastructure/test_customization_template.py

```diff
@@ -63,7 +65,7 @@ def test_pxe_image_type_required_error_validation(collection):
             script_data='Testing the script')
 
 
-@pytest.mark.uncollectif(lambda: version.current_version() < '5.9')
```

Comment:
there's a BZ 1449116 for this test case. it was created against 5.8 but got fixed in 5.9. It has just got 5.8z flag - fix will be copied into 5.8. Also 5.7 wasn't affected but issue with duplicates - taking all this into account - better to leave this until in gets fixed in 5.8 too